app.js

```diff
@@ -42,20 +41,21 @@ app.get("/edit/:id", controller.edit);
 app.get("/admin", controller.admin);
 app.get("/submit", controller.submit);
 app.get("/tag/:tag", controller.filter_by_tag);
-app.post("/post", controller.create);
 app.get("/search/:tag", controller.search_tags);
 app.get("/searchOne/:id", controller.search_findOne);
-app.post("/update", controller.update_project);
-app.post("/approve", controller.approve_project);
 app.get("/image_upload/:id", controller.upload_page);
-app.post("/upload", controller.upload);
 app.get('/faq', controller.faq);
 app.get("/projects", controller.projects);
 app.get("/startups", controller.startups);
 app.get("/create_account_page", controller.create_account_page);
-app.post("/create_account", controller.create_account);
 app.get("/login", controller.login);
-app.post("/check_login", controller.check_login);
 app.get("/logout", controller.logout);
 
+app.post("/update", controller.update_project);
+app.post("/approve", controller.approve_project);
+app.post("/upload", controller.upload);
+app.post("/create_account", controller.create_account);
+app.post("/post", controller.create);
+app.post("/check_login", controller.check_login);
```

Comment:
@adelq just separating get and post requests for clarity.  Please feel free to revert if you prefer it the old way.